src/Extension/FluentExtension.php

```diff
@@ -1055,6 +1055,14 @@ public function LocaleInformation($locale = null)
             $localeObj = Locale::getDefault();
         }
 
+        if (!$locale && !$localeObj) {
+            // There is no default locale, this can happen when if fluent is installed though locales have been setup
```

Comment:
```suggestion
            // There is no default locale, this can happen if no locales have been setup
```
I assume you meant that NO locales have been set up.
The fact that fluent is installed is a given when fluent code is executed, so removed that for brevity.

Reply:
Updated